artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -151,11 +155,16 @@ private void printStats(String result) {
       }
 
       JsonObject queuesAsJsonObject = JsonUtil.readJsonObject(result);
-      JsonArray array = (JsonArray) queuesAsJsonObject.get("data");
+      int count = queuesAsJsonObject.getInt("count");
+      JsonArray array = queuesAsJsonObject.getJsonArray("data");
 
       for (int i = 0; i < array.size(); i++) {
          printQueueStats(array.getJsonObject(i));
       }
+
+      if (maxRows == null && count > printingMaxRows) {
```

Comment:
just count > maxRows

Reply:
I would not to bore users with a warning if they set the maxRows parameter. WDYT?

Reply:
i think its a) simpler to not discriminate b) even if i set 100 manually, it still useful to get an output to tell me whilst i set 100 it wasnt all the messages (and include the count in the output)

Reply:
If you limit a sql query you get a limited set of record and you don't get a warning but if you don't limit a sql query you should get all matching records.
I think that the warning could break a script that is processing the queue stat output, so I would print the total available queues only in verbose mode if the user set the maxRows. WDYT?

Reply:
this isnt a sql query though, its an admin tool, and like you said in sql if you dont limit you get all, thats not behaviour here, so its not the same at all.